api/src/main/java/org/openmrs/Order.java

```diff
@@ -54,6 +54,16 @@
 		RENEW
 	}
 	
+	/**
+	 * Requested by TRUNK-5400
+	 */
+	public enum FulfillStatus {
```

Comment:
Shouldn't the name be Fulfill**er**Status

Reply:
+1